postgres/src/test.rs

```diff
@@ -373,14 +431,15 @@ fn portal() {
         .bind("SELECT * FROM foo ORDER BY id", &[])
         .unwrap();
 
-    let rows = transaction.query_portal(&portal, 2).unwrap();
-    assert_eq!(rows.len(), 2);
-    assert_eq!(rows[0].get::<_, i32>(0), 1);
-    assert_eq!(rows[1].get::<_, i32>(0), 2);
+    let rows = transaction.query_portal(&portal, 3).unwrap();
+    assert_eq!(rows.len(), 3);
+    assert_eq!(rows[0].get::<_, i64>(0), 1);
+    assert_eq!(rows[1].get::<_, i64>(0), 2);
+    assert_eq!(rows[1].get::<_, i64>(0), 2);
```

Comment:
Should this line be the following?
```suggestion
    assert_eq!(rows[2].get::<_, i64>(0), 3);
```

Reply:
Yes, I'll fix that, thanks.